### PR TITLE
[MM-58397] Add calls offloader metrics, incl. cpu, memory, and live captions

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Adds an offloader row and a few panels for offloader health.
- Merge when Calls v0.28.0 is released

Screenshot:

CleanShot 2024-06-13 at 09 23 50@2x

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-58397

### CODE REVIEW

File: grafana/mattermost-calls-performance-monitoring.json [hunk 1]
```diff
@@ -2812,99 +2819,774 @@
...
+        "multi": true,
+        "name": "offloader",
+        "options": [],
+        "query": {
+          "qryType": 1,
+          "query": "label_values(instance)",
+          "refId": "PrometheusVariableQueryEditor-VariableQuery"
+        },
+        "refresh": 1,
+        "regex": "(.*offloader.*):$offloader_metrics_port",
```
Comment (cpoile) [inline grafana/mattermost-calls-performance-monitoring.json]: Open to better solutions for this.

Comment (streamer45) [reply grafana/mattermost-calls-performance-monitoring.json]: There is no offloader specific metrics endpoint as far as I remember as we re-route everything through the plugin so I don't think we need this. We can hardcode `9100` in the system queries as it's the standard port for node exported metrics.

Comment (cpoile) [reply grafana/mattermost-calls-performance-monitoring.json]: Sounds good, removed it.

**Review by streamer45 — APPROVED** [review]
Looks great!

**Comment by streamer45** [comment]
@cpoile This can be merged now.

**Pull request merged**